flocker/docs/_version.py

```diff
@@ -163,3 +169,37 @@ def is_pre_release(version):
             and parsed_version.weekly_release is None
             and parsed_version.commit_count is None
             and parsed_version.dirty is None)
+
+
+def get_pre_release(version):
```

Comment:
Given a class with methods, why are you adding new functions? Why doesn't the admin script just use `FlockerVersion` instances directly?